examples/common/operator.yaml

```diff
@@ -1633,6 +1633,35 @@ spec:
                 - name
                 type: object
               type: array
+            upgrade:
```

Comment:
Same comments, perhaps we can copy the file not to manually replicate it?

Reply:
this file is an output of:
`kustomize build config/operator/default > examples/common/operator.yaml`